app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,15 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def add_room
+    @guest = Guest.find(params[:id])
+    room_id = params[:room_id]
+
+    room = Room.find_by(id: room_id)
+    @guest.rooms << room if room
```

Comment:
using `if room` is a nice way to verify the validity of the ID - not required but nice!